docs/source/federation/other-servers.md

```diff
@@ -12,5 +12,6 @@ The following is a list of known open source GraphQL server libraries that have
 | PHP         | wp-graphql   | [wp-graphql-federation](https://github.com/wp-graphql/wp-graphql-federation)     |
 | Python      | graphene     | [graphene-federation](https://github.com/preply/graphene-federation)             |
 | Ruby        | graphql-ruby | [apollo-federation-ruby](https://github.com/Gusto/apollo-federation-ruby)        |
+| Go          | gqlgen       | [gqlgen](https://github.com/99designs/gqlgen/tree/master/plugin/federation)      |
```

Comment:
Can you sort this new addition into the list alphabetically? (i.e. move it to the top)

Reply:
Done. Thanks for the comment 🙇